libs/ng-utils/src/lib/use-http-image-source/use-http-image-source.pipe.ts

```diff
@@ -39,6 +45,27 @@ export class UseHttpImageSourcePipe implements PipeTransform, OnDestroy {
     this.setUpSubscription();
   }
 
+  /**
+   * Returns a valid url path (blob path, loadingImagePath or errorImagePath) or null based on image fetching state.
+   *
+   * @example
+   * // Inject UseHttpImageSourcePipe into module
+   * sourcePipe = module.inject(UseHttpImageSourcePipe);
+   *
+   * // subscribing to the observable will create the database
```

Comment:
`// subscribing to the observable will create the database` is not relevant here